### ISSUE TITLE
Ability to output bare ID of an execution

### ISSUE THREAD

**Issue by jtopjian** [StackStorm/st2#1711]
Per discussion in IRC, I'd like to be able to print the bare ID (perhaps any field) when listing the execution history. My current method of doing this is:

``` shell
$ st2 execution list --action pack.foo --status succeeded -n 1 -a id -j | grep id | awk '{print $2}' | tr -d \"
```

But a native way, without relying on `awk` and `tr` would be nice. :smile:

**Comment by manasdk** [comment]
Agreed. Will try to see what can be supported to allow getting to the ID quicker.

**Comment by Kami** [comment]
Yep, I also agree.

Ideally, we would come up with some pattern which works consistently across the CLI. Maybe something like `--attr <attribute name> --raw`?

**Comment by lakshmi-kannan** [comment]
--attr is already available. --raw is currently used in executions to mean something else.

**Comment by lakshmi-kannan** [comment]
@Kami I tried working on this and got super confused with https://github.com/StackStorm/st2/blob/master/st2client/st2client/commands/action.py#L244

From the description of the commands, it's not clear what's the intended difference between `raw` and `tasks`. Perhaps you do?

**Comment by Kami** [comment]
@lakshmi-kannan "--raw" flag means don't show / include workflow tasks in the output - just show the top-level result and "--tasks" is the opposite.

We detect if a particular execution is a workflow and default to printing the child tasks. If you don't want this behavior you can use the "--raw" flag. On the other hand, if we don't correctly detect a workflow action you can force the child tasks to be printed using "--tasks" flag.

**Comment by Kami** [comment]
> --attr is already available. --raw is currently used in executions to mean something else.

Yeah, I'm aware of "--attr" and we can come up with something better than "--raw" for this use case :D

**Comment by Kami** [comment]
#3953 should now do the trick API wise, and --attr CLI wise.

**Comment by LindsayHill** [comment]
Resolved by #3953
